src/test/java/cucumber_runner/TestRunner.java

```diff
@@ -12,7 +12,7 @@
 @CucumberOptions(
         glue = { "stepDefinition", "hooks" },
         features = { "src/test/resources/features" },
-        tags = { "not @amiordan", "not @mraicov", "not @aciuvaga" },
+        tags = { "@Andrei_Ciuvaga" },
```

Comment:
revert this change and edit just your tag with `not` prefix as others don't need to run  your scenarios

Reply:
Done